coastlines/vector.py

```diff
@@ -183,6 +183,9 @@ def ocean_masking(ds, ocean_da, connectivity=1, dilation=None):
         pixels as True.
     """
 
+    # Squeeze out dimension as required
+    ds = ds.squeeze()
```

Comment:
Required due to changes to `.groupby() in recent versions of `xarray` (it no longer applies `.squeeze()` to each group automatically)